### DISCUSSION TITLE
Error: Command failed: git clone --depth 1 https://github.com/walletconnect/create-wc-dapp/

### DISCUSSION THREAD

**Discussion by Densito78** [WalletConnect/walletconnect-monorepo#4463] [Developer Support]
Hello,
I tried to execute npx create-wc-dapp@latest -y web3modal,
but got this error:
Error: Command failed: git clone --depth 1 https://github.com/walletconnect/create-wc-dapp/

**Comment by glitch-txs** [comment]
hi, the CLI is currently outdated, please use the custom configuration 🙏
